code/modules/mob/living/simple_animal/abnormality/aleph/space_lady.dm

```diff
@@ -13,7 +13,6 @@
 	can_breach = TRUE
 	threat_level = ALEPH_LEVEL
 	start_qliphoth = 2
-	ranged = 1
```

Comment:
Is this a accidental removal or part of the fix?

Reply:
This part of the fix. Space lady  has ranged = TRUE a few lines below.